src/integratedServer/browserfsShared.ts

```
@@ -0,0 +1,409 @@
+          fs: 'HTTPRequest',
+          options: {
+            index,
+            baseUrl: fsState.remoteBackendBaseUrl
+          }
+        }
+      },
+    }, (e) => {
+      if (e) throw e
+      resolve()
+    })
+  })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)